src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml

```diff
@@ -1,234 +0,0 @@
-
-import QtQuick 2.5
-
-import Mozilla.Shared 1.0
-import Mozilla.VPN 1.0
-import components 0.1
-import compat 0.1
-
-Rectangle {
-    property bool isOpen: false
-    property bool isTransitioning: false
-    property int transitionDuration: 750
-
-    id: root
-
-    clip: true
-    color: MZTheme.colors.primary
-    opacity: 0
-    state: "closed"
-    states: [
-        State {
-            name: "closed"
-            when: !isOpen && !isTransitioning
-
-            PropertyChanges {
-                target: root
-                opacity: 0
-                restoreEntryValues: false
-            }
-            StateChangeScript {
-                script: VPNConnectionBenchmark.reset();
-            }
-        },
-        State {
-            name: "opening"
-            when: isOpen && isTransitioning
-
-            PropertyChanges {
-                target: root
-                opacity: 1
-                visible: true
-                restoreEntryValues: false
-            }
-        },
-        State {
-            name: "closing"
-            when: !isOpen && isTransitioning
-            StateChangeScript {
-                script: if(root.isTransitioning) root.opacity = 0
-            }
-        },
-        State {
-            name: "open-loading"
-            when: VPNConnectionBenchmark.state === VPNConnectionBenchmark.StateRunning
-                && isOpen
-
-            PropertyChanges {
-                target: root
-                opacity: 1
-                visible: true
-                restoreEntryValues: false
-            }
-        },
-        State {
-            name: "open-ready"
-            when: VPNConnectionBenchmark.state === VPNConnectionBenchmark.StateReady
-                && isOpen
-
-            PropertyChanges {
-                target: root
-                opacity: 1
-                visible: true
-                restoreEntryValues: false
-            }
-        },
-        State {
-            name: "open-error"
-            when: VPNConnectionBenchmark.state === VPNConnectionBenchmark.StateError
-                && isOpen
-
-            PropertyChanges {
-                target: root
-                opacity: 1
-                visible: true
-                restoreEntryValues: false
-            }
-        }
-    ]
-    width: parent.width
-
-    onIsOpenChanged: () => {
-        // Start opening/closing transition
-        isTransitioning = true;
-
-        if (VPNConnectionBenchmark.state === VPNConnectionBenchmark.StateInitial) {
-            VPNConnectionBenchmark.start();
-        } else {
-            VPNConnectionBenchmark.reset();
-        }
-
-        timer.setTimeout(function() {
-            // Finished opening/closing transition
-            isTransitioning = false;
-        }, transitionDuration);
-    }
-
-    onStateChanged: () => {
-        if (state === "open-loading") {
-            Glean.impression.speedTestLoadingScreen.record({
```

Comment:
I _think_ we can remove these pings now. @mcleinman is there a reason to keep these in `metrics.yaml` if we are no longer sending?

Reply:
No, we can remove the metrics. Pedantic, but: I don't believe we can remove any pings (the sending of a group of metrics), but the metrics themselves can be removed here and in `metrics.yaml`.

Reply:
thank you! updated my comment to conform to appropriate nomenclature!

Reply:
I have removed the metrics